clusters/build-clusters/build03/quayio-pull-through-cache/quayio-pull-through-cache.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: quayio-pull-through-cache
+    namespace: ci
+  spec:
+    selector:
+      app: quayio-pull-through-cache
+    ports:
+    - port: 5000
+      targetPort: 5000
+    type: ClusterIP
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: quayio-pull-through-cache
+    namespace: ci
+    labels:
+      app: quayio-pull-through-cache
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: quayio-pull-through-cache
+    template:
+      metadata:
+        labels:
+          app: quayio-pull-through-cache
+      spec:
+        containers:
+        - name: quayio-cache
+          image: docker.io/registry:2.8.1
```

Comment:
This will create a problem since docker.io has a `pull` limit request. We could push it in our quay.io repo and use that one instead.